src/patternfly/components/Button/button.scss

```
@@ -879,6 +881,8 @@
     --#{$button}--PaddingInlineEnd: var(--#{$button}--m-circle--PaddingInlineEnd);
     --#{$button}--PaddingBlockEnd: var(--#{$button}--m-circle--PaddingBlockEnd);
     --#{$button}--PaddingInlineStart: var(--#{$button}--m-circle--PaddingInlineStart);
+    aspect-ratio: 1;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)